### PR TITLE
change: [M3-8075] - "Create Volume" button text changed to "Add Volume".

### PR DESCRIPTION
**Pull request by dchyrva**
## Description 📝
"Create Volume" button text changed to "Add Volume".

## Changes  🔄
"Create Volume" button text changed to "Add Volume".

### CODE REVIEW

**Comment by dchyrva** [comment]
Hello @jdamore-linode,

Thank you very much for your comments, they should be resolved now.

However, I can see that "continuous-integration/cloud-manager-tests" step is still failing in the pipeline,
could you please advice, is this something that should be fixed locally?

Thank you very much in advance.

**Review by bnussman-akamai — APPROVED** [review]
Nice job! 👏 🎉

File: packages/manager/.changeset/pr-10808-changed-1724315097131.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Changed
+---
+
+"Create Volume" button text changed to "Add Volume" ([#10808](https://github.com/linode/manager/pull/10808))
```
Comment (mjac0bs) [inline packages/manager/.changeset/pr-10808-changed-1724315097131.md:5]:
```suggestion
"Create Volume" button text to "Add Volume" ([#10808](https://github.com/linode/manager/pull/10808))
```
From our [changeset best practices](https://linode.github.io/manager/CONTRIBUTING.html#writing-a-changeset): 
> Use a consistent tense in all changeset entries. We have chosen to use imperative (present) tense. (This follows established [git commit message best practices](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).)
> Avoid starting a changeset with the verb "Add", "Remove", "Change" or "Fix" when listed under that respective Added, Removed, Changed or Fixed section. It is unnecessary repetition.

File: packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts [hunk 2]
```diff
@@ -164,27 +165,29 @@ describe('volume create flow', () => {
           localStorageOverrides: pageSizeOverride,
         });
 
-        // Click "Create Volume" button, fill out and submit volume create drawer form.
-        fbtClick('Create Volume');
+        // Click "Add Volume" button, fill out and submit volume create drawer form.
+        cy.findByText('Add Volume').click();
```
Comment (mjac0bs) [inline packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts:169]: @dchyrva The reason for the failure was the same as `create-volume.smoke.spec.ts`, just happening in another test. This line was the fix. All tests related to this PR are passing locally now, so this should be set to merge.

The rest of the changes in the file are 1:1 swaps of some deprecated Cypress functions for newer ones.

**Review by mjac0bs — APPROVED** [review]

**Pull request merged**